docs/sources/developers/plugins/working-with-data-frames.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Working with data frames
+title: Work with data frames
 ---
 
-# Working with data frames
+# Work with data frames
```

Comment:
Working feels more correct here, interested to know why the change?

Why don't we just merge this content into data-frames.md?